roles/installer/defaults/main.yml

```diff
@@ -111,6 +111,8 @@ redis_image: docker.io/redis
 redis_image_version: latest
 postgres_image: postgres
 postgres_image_version: 12
+init_container_image: quay.io/centos/centos
```

Comment:
I agree with the approach of this PR, and it works, but is using this image OK @wenottingham @Spredzy

Reply:
I would assume in other places this would be overriden. @tchellomello ?

Reply:
Yes. The variable can be override.